src/rgw/rgw_sal_rados.cc

```diff
@@ -1420,6 +1427,18 @@ void RadosStore::finalize(void)
     rados->finalize();
 }
 
+void RadosStore::register_admin_apis(RGWRESTMgr_Admin* mgr)
+{
+  mgr->register_resource("bucket", new RGWRESTMgr_Bucket);
```

Comment:
what bucket apis are rados-specific? maybe the link/unlink stuff?

Reply:
Currently link, unlink, and get stale instances.  At some point, we can fix/break these up.